src/wireviz/DataClasses.py

```diff
@@ -28,14 +28,17 @@ def __post_init__(self):
                 else     "true" # When only one dimension is specified.
 
         if self.fixedsize is None:
-            self.fixedsize = self.width or self.height
-
-        if self.height:
-            if not self.width:
-                self.width = self.height # Assuming 1:1 aspect ratio for now.
-        else:
-            if self.width:
-                self.height = self.width # Assuming 1:1 aspect ratio for now.
+            # Normally equal to (self.width or self.height), but the user might override self.scale.
+            self.fixedsize = str(self.scale).lower() in ["true", "both"]
```

Comment:
I also found a draw-back with the new default value: The `fixedsize` might now be set default True even if no dimension is specified. That will result in an error from Graphviz. I should change it to a combination of the two conditions to avoid that:
```
# Avoid default True if the user overrides self.scale to a different value.
self.fixedsize = (self.width or self.height) and str(self.scale).lower() in ["true", "both"]
```